api/v1/calico_kubecontrollers_types.go

```diff
@@ -59,6 +59,12 @@ type CalicoKubeControllersDeploymentPodSpec struct {
 	// WARNING: Please note that this field will modify the default calico-kube-controllers Deployment nodeSelector.
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
 
+	// TopologySpreadConstraints describes how a group of pods ought to spread across topology
+	// domains. Scheduler will schedule pods in a way which abides by the constraints.
+	// All topologySpreadConstraints are ANDed.
+	// +optional
+	TopologySpreadConstraints []v1.TopologySpreadConstraint `json:"topologySpreadConstraints,omitempty"`
```

Comment:
Does it make sense to have this on kubecontrollers since we only allow one instance?